include/one_point_cutscene.h

```diff
@@ -0,0 +1,18 @@
+#ifndef ONE_POINT_CUTSCENE_H
+#define ONE_POINT_CUTSCENE_H
+
+#include "ultra64.h"
+
+struct Actor;
+struct PlayState;
+
+s16 OnePointCutscene_Init(struct PlayState* play, s16 csId, s16 timer, struct Actor* actor, s16 parentCamId);
+s16 OnePointCutscene_EndCutscene(struct PlayState* play, s16 subCamId);
+s32 OnePointCutscene_Attention(struct PlayState* play, struct Actor* actor);
```

Comment:
This is just a random side note but I really dislike lately that we called this "one point cutscene"
Kinda wish we stuck with OnePointDemo until we decide to give it a full rename with a more fitting name (if there is one)

Anyways not expecting you to change anything here just rambling

Reply:
yeah I don't like this name either